pulsar-sql/presto-server/src/resources/etc/catalog/pulsar.properties

```diff
@@ -0,0 +1,5 @@
+connector.name=pulsar
```

Comment:
any reason why we can't place /etc/catalog/pulsar.properties to conf/presto/catalog/pulsar.properties? then we have a central place for managing all configuration files for pulsar components.

Reply:
basically `/etc` -> `conf/presto`

Reply:
good idea!  I was just trying to keep the original presto project structure